python/lsst/ts/electrometer/csc.py

```diff
@@ -1,7 +1,8 @@
 from lsst.ts import salobj
 from lsst.ts.idl.enums import Electrometer
 import pathlib
-from . import model
+import asyncio
+from . import controller
```

Comment:
Please consider using our PEP8 and our standard import order: system libraries first, then a blank line, then 3rd party libraries, then T&S libraries.